build-docker-container.sh

```diff
@@ -1,3 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
-docker build --rm -t ddoyle/rhoas-devsandbox-workshop-tools .
\ No newline at end of file
+# set -uo pipefail
+
+rhoas_version=$TAG
```

Comment:
```suggestion
TAG=${TAG:-latest}
rhoas_version=$TAG
```

Reply:
Ah.. that tag is actual version of CLI not tag of the image.

Reply:
Done